setup.cfg

```diff
@@ -1,3 +1,17 @@
-[templer.local]
-template = plone_basic
+[isort]
```

Comment:
@ale-rt I guess we kind of have to decide where we want to keep configuration like this. So far some of this is in p.r.codeanalysis. I am ok moving into a more pythonic direction. Though, then we should reflect that in the p.r.codeanalysis docs.